src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,85 @@
+package nextstep.courses.domain;
+
+import nextstep.payments.domain.Payment;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import static nextstep.courses.domain.SessionPayType.PAID;
+import static nextstep.courses.domain.SessionState.PREPARING;
+import static nextstep.courses.domain.SessionState.RECRUITING;
+
+public class Session {
+    private Long id;
+    private Course course;
+    private LocalDate startDate;
+    private LocalDate endDate;
```

Comment:
```suggestion
    private LocalDate startDate;
    private LocalDate endDate;
```

비슷한 성격의 데이터로 보이는군요. 인스턴스 변수를 줄여볼수 있지 않을까요?

Reply:
요구사항에서 아직 사용하는 곳이 없어, 우선 inner class로 만들어보겠습니다!